components/rootless_studio/tests/dejagnu/studio/secrets_are_passed.exp

```diff
@@ -0,0 +1,55 @@
+# TODO: Pass in an argument for the Docker image
+
+# Verify that environment variables prefixed with HAB_STUDIO_SECRET_ do
+# in fact make it into an interactive Docker studio.
+
+exit -onexit {
+    # Be sure to always shut the Studio down; otherwise we'll leave
+    # the container running.
+    send "exit\r"
+    wait 2
+}
+
+# Define this as a variable for convenience and readability.
+set studio_prompt "default:/src:0\]# "
+
+# TODO (CM): This is currently a bit of a hack, in that it assumes
```

Comment:
I think this is because we're missing `HAB_NOCOLORING` here https://github.com/habitat-sh/habitat/blob/master/components/hab/src/command/studio/docker.rs#L83-L97